src/react/utils/useHydrateAtoms.ts

```diff
@@ -8,7 +8,7 @@ type AnyWritableAtom = WritableAtom<unknown, any[], any>
 const hydratedMap: WeakMap<Store, WeakSet<AnyWritableAtom>> = new WeakMap()
 
 export function useHydrateAtoms(
-  values: Iterable<readonly [AnyWritableAtom, unknown]>,
+  values: (readonly [AnyWritableAtom, unknown])[],
```

Comment:
Does this support `Map`?

Reply:
No. I didn't realize here accept a map

Reply:
#1898 will be adding a test with `Map`.

Reply:
Ok i will update the document instead of code